tests/wasmfs/wasmfs_create.c

```diff
@@ -47,13 +47,13 @@ int main() {
 
   // Try to create an existing file with O_EXCL and O_CREAT.
   errno = 0;
-  int fd2 = open("/dev/stdin", O_RDWR | O_CREAT | O_EXCL);
+  int fd2 = open("/test", O_RDWR | O_CREAT | O_EXCL, S_IRWXU);
```

Comment:
is the mode addition here necessary?

Reply:
Not entirely, but the spec says that if you use O_CREAT, you _must_ provide a mode. If you don't provide a mode, it ends up taking random stack bits as the mode, which doesn't happen to matter here, but seems bad in general.

Reply:
Ah, sounds good, I didn't know that.